about.html

```diff
@@ -0,0 +1,119 @@
+          </li>
+          <li>
+            <a href="#">English</a>
+          </li>
+          <li>
+            <a href="#">My Page</a>
+          </li>
+          <li>
+            <a href="#">Logout</a>
+          </li>
+        </ul>
+        <i class="bi bi-x close-menu"></i>
+        <nav>
+          <a href="./index.html"><img src="./assets/images/logo-rg.svg" class="logo" alt="Roland Garros"></a>
+          <ul class="lato">
+            <li><a href="./about.html" class="active">About</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#">Tournament</a></li>
+            <li><a href="#">Tickets</a></li>
+            <li><a href="#">Store</a></li>
+            <li><a href="#" class="highlighted-menu">Roland Garros 2023</a></li>
+          </ul>
+        </nav>
+    </header>
+    <section id="about-headline">
+      <i class="bi bi-list" id="menu-button"></i>
+      <div class="lato breadcrumbs"></div>
+      <h3 class="lato">"Hello! Tennis Fans"</h3>
+      <h1 class="cocogoose">ROLAND GARROS 2023 <br>
+            THE MAIN TENNIS EVENT IN THE SEASON IS COMING
+            IN MAY 2023
+      </h1>
+      <p class="lato">The French Open, also known as Roland Garros, is one of the four 
+            Grand Slam tournaments in tennis. The tournament is named after Roland Garros, 
+            a French aviator and World War I hero. The tournament was first held in 1891 and 
+            was open only to French tennis players. In 1925, the tournament became open to 
+            international players and was held at the Stade Roland Garros, which is located in
+             the 16th arrondissement of Paris. Over the years, the tournament has changed in many ways. 
+             In 2019, floodlights were added so that games could be played in the evening.
+          <br> <br>
+          In the 1980s, the tournament saw the emergence of several legendary players, 
+          such as Ivan Lendl and Chris Evert, who won multiple titles at the tournament. 
+          However, it was in the 2000s that a new era of dominance began with the 
+          emergence of one of the greatest players of all time, Rafael Nadal. Nadal has 
+          won a record 13 French Open titles, and his dominance on the clay courts of 
+          Roland Garros is unmatched. He has won the tournament almost every year since 
+          his first win in 2005, except for a few years when he was either injured or 
+          unable to compete. Nadal’s success at the tournament has solidified his place 
+          in tennis history and has made him a household name around the world. His rivalry 
+          with Roger Federer and Novak Djokovic has also added to the tournament’s appeal, 
+          making it a highly anticipated event every year.
+      </p>
+      <p class="lato about-info">Please contact us per email for any further questions about Roland Garros Tournament!</p>
+      <h5 class="lato">info@rolandgarros.com</h5>
+    </section>
```

Comment:
- Kindly make sure that you add the appropriate  background images to this section to make it look luck the design provided as required in the project :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/235263277-f506d594-8cdb-4baf-a461-46210011c1f1.png)

## The design provided 

![image](https://user-images.githubusercontent.com/95297251/235263399-599d04f5-d47d-44a1-89d7-7b5fe0b6aace.png)

Reply:
![issue-03](https://user-images.githubusercontent.com/86327867/235285878-bdd81ea2-b9dc-45d5-9713-02a2828b7f49.jpg)

**Fixed the background image in about.html** page.